tests/test_comb.c

```diff
@@ -111,45 +113,46 @@ static int action_start(struct nmd_ctx *s, int opt_test_flags)
 static int action_middle(struct nmd_ctx *s, int opt_test_flags)
 {
     int ret;
-    struct nmd_frame *f0 = nmd_get_frame(s, 30.0);
-    struct nmd_frame *f1 = nmd_get_frame(s, 30.1);
-    struct nmd_frame *f2 = nmd_get_frame(s, 30.2);
-    struct nmd_frame *f3 = nmd_get_frame(s, 15.0);
-    struct nmd_frame *f4 = nmd_get_next_frame(s);
-    struct nmd_frame *f5 = nmd_get_next_frame(s);
+    struct nmd_frame *frames[6] = {NULL};
```

Comment:
I'd rather use `{0}` here because it implies that the code is equivalent to `{NULL, NULL, NULL, ...}` while it's actually `{NULL, 0, 0, ...}`

Reply:
Changed.